thunder/core/trace.py

```diff
@@ -572,3 +572,16 @@ def _set_execution_file(path: str) -> None:
 
 def _get_execution_file() -> None | str:
     return _execution_file.get()
+
+
+#
+# Container for the two/three types of traces, plus extra tracked data
+#
+
+
+class JitResults:
+    def __init__(self, prologue: TraceCtx, computation: TraceCtx, epilogue: TraceCtx | None, history: list):
+        self.prologue_trace = prologue
+        self.computation_trace: TraceCtx = computation
+        self.epilogue_trace = epilogue
+        self.history = history
```

Comment:
It's a nitpick, but naming is important. History is awefully generic, can you make it `interpretation_log` or somesuch instead?